### PR TITLE
Update list of data-aggregator-breaches

### LINKED ISSUE #950: Apollo is categorized as website breach. Should be data aggregator.
Apollo appears as a website breach on https://fx-breach-alerts.herokuapp.com/breach-details/Apollo.

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #950 /cc @betsymi 

Not a fan of hardcoding a list in one of the files, but I'm not sure it's any better in the AppConstants or package.json or server.js file.

### CODE REVIEW

**Comment by pdehaan** [comment]
Pre PR:

| `breach.Name` | Category |
|:--------------|:---------|
| [Apollo](https://monitor.firefox.com/breach-details/Apollo) | website-breach
| [ElasticsearchSalesLeads](https://monitor.firefox.com/breach-details/ElasticsearchSalesLeads) | data-aggregator-breach
| [Estonia](https://monitor.firefox.com/breach-details/Estonia) | data-aggregator-breach
| [Exactis](https://monitor.firefox.com/breach-details/exactis) | data-aggregator-breach
| [MasterDeeds](https://monitor.firefox.com/breach-details/MasterDeeds) | data-aggregator-breach
| [YouveBeenScraped](https://monitor.firefox.com/breach-details/youvebeenscraped) | data-aggregator-breach

File: template-helpers/breach-detail.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ function getVars(args) {
 // How to dynamically detect Data Aggregator breaches?
 
 function getBreachCategory(breach) {
-  if (["Exactis"].includes(breach.Name)) {
+  if (["Exactis", "Apollo", "YouveBeenScraped", "ElasticsearchSalesLeads", "Estonia", "MasterDeeds"].includes(breach.Name)) {
```
Comment (groovecoder) [inline template-helpers/breach-detail.js]: I personally think Adapt, Verifications.io, Data & Leads, and others are all data aggregators. How comprehensive do we want/need this list to be? (cc @betsymi )

Comment (pdehaan) [reply template-helpers/breach-detail.js]: I prefer to err on the side of comprehensive. This makes our current data aggregator breach list:

```js
  "dataAggregatorBreaches": [
    "Adapt",
    "Apollo",
    "DataAndLeads",
    "ElasticsearchSalesLeads",
    "Estonia",
    "Exactis",
    "MasterDeeds",
    "VerificationsIO",
    "YouveBeenScraped"
  ],
```

Comment (betsymi) [reply template-helpers/breach-detail.js]: @pdehaan @groovecoder Thank you for the work you've done to aggregate the data aggregators.

**Pull request merged**